app/models/location.rb

```diff
@@ -40,6 +40,7 @@ class Location < ApplicationRecord
       filtered_records = filtered_records
       .left_outer_joins(:location_deactivation_periods)
       .joins(scheme: [:owning_organisation])
+      .left_outer_joins(scheme: :scheme_deactivation_periods)
```

Comment:
Location status now also relies on the status of the scheme it's deactivation periods.